src/v3/providers/coingecko/coingecko.ts

```diff
@@ -127,6 +132,67 @@ automatedTokenMappingsSyncDoc.onChange(automatedMappings => {
   }
 })
 
+const coingeckoToCrossChainMapping = async (
+  coingeckoAssets: ReturnType<typeof asCoingeckoAssetResponse>,
+  tokenTypes: StringNullMap
+): Promise<CrossChainMapping> => {
+  const invertPlatformMapping: { [key: string]: string } = {}
+  for (const [key, value] of Object.entries(coingeckoPlatformIdMapping)) {
```

Comment:
Do this for loop once in `defaultPluginIdMapping.ts` it's 100% static

Reply:
Should have been platformIdMappingSyncDoc which is dynamic